frontend/coprs_frontend/coprs/templates/layout.html

```diff
@@ -114,15 +114,29 @@
                   </dd>
                 </dl>
               </div>
-              <div class="col-sm-3">
+              <div class="col-sm-2">
                 <dl>
                   <dt><h3>Powered by</h3></dt>
                   <dd>
                     <ul>
                       <li><a href="https://www.python.org">Python</a></li>
-                      <li><a href="http://www.openstack.org">OpenStack</a></li>
                       <li><a href="https://www.patternfly.org">PatternFly</a></li>
                       <li><a href="https://github.com/release-engineering/dist-git">Dist Git</a></li>
+                      <li><a href="https://github.com/praiskup/resalloc">Resalloc</a></li>
+                    </ul>
+                  </dd>
+                </dl>
+              </div>
+              <div class="col-sm-2">
+                <dl>
+                  <dt><h3>Sponsored by</h3></dt>
+                  <dd>
+                    <ul>
+                      <li><a href="https://www.redhat.com">Red Hat</a></li>
+                      <li><a href="https://www.fedoraproject.org">Fedora</a></li>
```

Comment:
Not sure about Fedora ... Fedora Infra is great at taking care of us, but I don't know if that means we are sponsored by Fedora. Personally, I'd probably move Fedora to the "Powered by" column.

Reply:
I would vote to keep it there. Without Fedora... there would be no Copr.